release-notes/swan-lake-alpha4-release-note.md

```diff
@@ -56,8 +56,65 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 #### New Features
 
+##### Support for Relational Expressions With All Ordered Types
+
+Relational expressions (`<`, `>`, `<=`, and `>=`) are supported with all [ordered types](https://ballerina.io/spec/lang/draft/v2020-12-17/#ordering). The static type of both operands must belong to the same ordered type.
+
+##### Support for inferring the `typedesc` argument of a dependently-typed function from the contextually expected type
+
+When the default value of a `typedesc` parameter of a dependently-typed function is `<>` and an argument is not provided for the parameter when calling the function, the argument will be inferred from the contextually expected type of the function call.
+```ballerina
+function func(typedesc<anydata> td = <>) returns td = external;
+
+public function main() {
+    // The argument for `td` is inferred to be `int`.
+    int value = func();
+}
+```
+
 #### Improvements
 
+##### Dependently-typed lang library functions that infer the argument from the contextually expected type
+
+The `cloneWithType`, `fromJsonWithType`, `fromJsonStringWithType`, and `ensureType` lang library functions are dependently-typed functions for which the `typedesc` argument will be inferred from the contextually-expected type if it is not passed as an argument.
+
+```ballerina
+import ballerina/io;
+
+type Person record {|
+    string name;
+    int age;
+|};
+
+public function main() {
+    map<anydata> anydataMap = {name: "Amy", age: 30};
+
+    // The `typedesc` argument is inferred to be `Person`
+    // based on the contextually expected type.
+    Person|error result = anydataMap.cloneWithType();
+    io:println(result is Person); // Prints `true`.
+}
+```
+
+##### The return type of `lang.value:cloneReadOnly` is a subtype of `readonly`
```

Comment:
```suggestion
##### The Return Type of `lang.value:cloneReadOnly` is a Subtype of `readonly`
```
Headings can't be complete sentences. This should either be rephrased or we can make it a bullet point.

Reply:
Updated in 37da79a.